src/ip_station_points_grid_mod.F90

```diff
@@ -1,11 +1,11 @@
 !> @file
-!> @brief ???
+!> @brief Interpolate gridded data to a series of station points.
 !>
-!> @author Kyle Gerheiser @date ???
+!> @author Kyle Gerheiser @date 7/21/21
 
-!> @brief ???
+!> @brief Interpolate gridded data to a series of station points.
```

Comment:
I take it files and modules get "@brief", subroutines do not?

Reply:
Yes, correct. The first sentence of the subprogram is automatically taken as the breif description. Unfortunately the same is not true for files and moduies so we need to use the tag there.